bom/pom.xml

```diff
@@ -545,6 +545,11 @@
         <artifactId>infinispan-client-hotrod</artifactId>
         <version>${infinispan.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.infinispan</groupId>
+        <artifactId>infinispan-query-dsl</artifactId>
```

Comment:
what do we need this for?

Reply:
see my other comment below

Reply:
IMHO we should define the scope here as `test` already to make it clear that we use this for testing only

Reply:
Ok, I've added the scope here.